ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
I could be wrong, `ORK_LOG_LEVEL_WARNING` is the default value (fallback) ? If the developer leave `Preprocessor Macro` blank? For example in the release config? 
<img width="418" alt="screen shot 2015-09-23 at 2 00 49 pm" src="https://cloud.githubusercontent.com/assets/11466704/10058660/870173d8-61fb-11e5-95c6-d0e8fec920f6.png">